src/data/coins.json

```diff
@@ -2477,7 +2477,12 @@
     ],
     "misc": [
         {
-            "blockchain_link": null,
+            "blockchain_link": {
```

Comment:
do not modify this file manually, use [this steps](https://github.com/trezor/connect/blob/develop/docs/supported-coins.md) to update coins.json

Reply:
Thanks! https://github.com/trezor/trezor-firmware/pull/1673